lib/active_merchant/billing/gateways/borgun.rb

```diff
@@ -180,19 +180,30 @@ def headers
         }
       end
 
-      def build_request(action, post)
+      def build_request(action, post, options={})
         mode = action == 'void' ? 'cancel' : 'get'
         xml = Builder::XmlMarkup.new indent: 18
         xml.instruct!(:xml, version: '1.0', encoding: 'utf-8')
         xml.tag!("#{mode}Authorization") do
           post.each do |field, value|
             xml.tag!(field, value)
           end
+          build_airline_xml(xml, options[:passenger_itinerary_data]) if options[:passenger_itinerary_data]
```

Comment:
This seems a little hinky, but refactoring how XML is generated felt out of scope.

Reply:
I'm okay with this hinkiness 👍